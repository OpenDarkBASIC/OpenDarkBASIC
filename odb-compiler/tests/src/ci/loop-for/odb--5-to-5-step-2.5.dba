for x# = -5 to 5 step 2.5
    print stdout x#
next x#
