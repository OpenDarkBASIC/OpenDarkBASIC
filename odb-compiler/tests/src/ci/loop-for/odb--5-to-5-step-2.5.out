"-5.000000\n-2.500000\n0.000000\n2.500000\n5.000000\n"
